.eslintrc.js

```diff
@@ -21,5 +22,6 @@ module.exports = {
     'react'
   ],
   'rules': {
+    'react/prop-types': 'off'
```

Comment:
Once eslint started running again, I got a bunch of prop-types errors.  IMO the solution with [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) is kinda clunky.  Since we don't have a ton of React code, I just disabled the check.  But I'm happy to revisit this.

Reply:
I think this okay